locations/ska95.yml

```diff
@@ -55,7 +55,7 @@ networks:
       ska95-core: 1
 
   - vid: 41  # Private LAN where private Internet is connected
-    role: ext
+    role: uplink
```

Comment:
In this case it's really simply an external private network that is connected to the core so the antennas could be managed from there. No uplink is done via that network. This should not be changed. There was a discussion about this in the emma chat. So we still have the ext role, but for uplinks we use uplink now.